### PR TITLE
[VIVO-1870] simplify menu page browse by vclass no entry translations

### PR DESCRIPTION
**Pull request by ghost**
[VIVO-1870](https://jira.lyrasis.org/browse/VIVO-1870)

Relates to https://github.com/vivo-project/Vitro/pull/182

@VIVO-project/vivo-committers

### CODE REVIEW

File: pt_BR/webapp/src/main/webapp/i18n/all_pt_BR.properties [hunk 1]
```diff
@@ -874,9 +874,8 @@ reordering_menus_failed = reordenação de itens de menu falhou.
 
 page = página
 view_page = Ver página
-= of_the_results dos resultados
```
Comment (ghost) [inline pt_BR/webapp/src/main/webapp/i18n/all_pt_BR.properties:877]: Addition fix here.

File: en_CA/webapp/src/main/webapp/i18n/all_en_CA.properties [hunk 2]
```diff
@@ -853,8 +853,7 @@ reordering_menus_failed = Reordering of menu items failed.
 page = page
 view_page = View page
 of_the_results = of the results
-there_are_no = There are no
-individuals_names_starting_with = individuals whose name starts with
+there_are_no_entry_starting_with = There are no entry starting with
```
Comment (awoods) [inline en_CA/webapp/src/main/webapp/i18n/all_en_CA.properties]: Should this be either "There are no entries..."? or "There is no entry..."?

Comment (ghost) [reply en_CA/webapp/src/main/webapp/i18n/all_en_CA.properties]: https://jira.lyrasis.org/secure/attachment/20055/JIRA%201870%20b.png

Comment (ghost) [reply en_CA/webapp/src/main/webapp/i18n/all_en_CA.properties]: Should I update?

Comment (awoods) [reply en_CA/webapp/src/main/webapp/i18n/all_en_CA.properties]: Probably. I think the grammar is wrong.

Comment (ghost) [reply en_CA/webapp/src/main/webapp/i18n/all_en_CA.properties]: Will update.

**Pull request merged**